fpm/Dockerfile.trusty

```diff
@@ -8,7 +8,7 @@ RUN apt-get update \
 COPY Gemfile /tmp
 COPY Gemfile.lock /tmp
 
-RUN gem install bundler \
+RUN gem install bundler -v 1.17.3\
```

Comment:
I'm guessing that `-v "< 2"` would also work, but still be flexible.

Reply:
Good suggestion. Amended and thank you.